Backend/tests/test_controllerResponse.py

```diff
@@ -0,0 +1,56 @@
+import unittest
+import sys
+from os import path
+sys.path.append( path.dirname( path.dirname( path.abspath(__file__) ) ) )
+from utils.ResponseCreation import *
+
+
+class ControllerResponseTests(unittest.TestCase):
+  @classmethod
+  def setUpClass(self):
+    self.response = ControllerResponse()
```

Comment:
I thought about it but decided it was one line and wasn't too big a deal. If you think it would give us a lot of benefit I can